runtime/astar/src/genesis_config.rs

```diff
@@ -40,25 +40,24 @@ pub fn default_config(para_id: u32) -> serde_json::Value {
     let dave = GenesisAccount::<sr25519::Public>::from_seed("Dave");
     let eve = GenesisAccount::<sr25519::Public>::from_seed("Eve");
 
-    let balances: Vec<(AccountId, Balance)> = vec![
-        (alice.account_id(), 1_000_000_000 * ASTR),
-        (bob.account_id(), 1_000_000_000 * ASTR),
-        (
-            TreasuryPalletId::get().into_account_truncating(),
-            1_000_000_000 * ASTR,
-        ),
-        (
-            CommunityTreasuryPalletId::get().into_account_truncating(),
-            1_000_000_000 * ASTR,
-        ),
-    ];
-
     let authorities = vec![&alice, &bob];
     let accounts = vec![&alice, &bob, &charlie, &dave, &eve]
         .iter()
         .map(|x| x.account_id())
         .collect::<Vec<_>>();
 
+    let balances = accounts
+        .iter()
+        .chain(
+            vec![
+                TreasuryPalletId::get().into_account_truncating(),
+                CommunityTreasuryPalletId::get().into_account_truncating(),
+            ]
+            .iter(),
+        )
+        .map(|x| (x.clone(), 1_000_000_000 * ASTR))
```

Comment:
Cloning is maybe unnecessary here (and below in local runtime) if x is consumed?